Makefile

```diff
@@ -513,6 +515,13 @@ run_deps: clone_deps
 		cd ${DEPS_DIR}/$$dep && $(MAKE) run; \
 	done
 
+# Run dependent projects in test mode (different project name to have different data and different common net name)
+run_test_deps: clone_deps
+	@for dep in ${DEPS} ; do \
+		cd ${DEPS_DIR}/$$dep && $(MAKE) run COMMON_NET_NAME=${COMMON_NET_NAME}_test COMPOSE_PROJECT_NAME=$${dep}_test MONGODB_EXPOSE=127.0.0.1:28017 REDIS_EXPOSE=127.0.0.1:6479; \
```

Comment:
I'm not sure why this is needed. In DOCKER_COMPOSE_TEST the shared services docker-compose.yml is included directly so that all needed dependencies start in the po_test project.

Reply:
Oups, my bad, I missed it for the docker compose !